cmd/pitr/recoverer/recoverer.go

```diff
@@ -272,6 +287,31 @@ func getLastBackupGTID(infoObj io.Reader) (string, error) {
 	return string(set), nil
 }
 
+func getDecompressedContent(infoObj io.Reader) ([]byte, error) {
+	tmpDir := os.TempDir()
+
+	cmd := exec.Command("xbstream", "-x", "--decompress")
+	cmd.Dir = tmpDir
+	cmd.Stdin = infoObj
+	var outb, errb bytes.Buffer
+	cmd.Stdout = &outb
+	cmd.Stderr = &errb
+	err := cmd.Run()
+	if err != nil {
+		return nil, errors.Wrapf(err, "xbsream cmd run. stderr: %s, stdout: %s", errb.String(), outb.String())
```

Comment:
not needed to call `.String()`, coz it will be called automatically

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/743/commits/55509d9d8f98677a13f4c5d3b91c3487c48ed868